corehq/apps/app_manager/templates/app_manager/ng_partials/form_summary_view.html

```diff
@@ -75,6 +75,12 @@ <h1 style="font-size: 1.8em;">{% trans "Form Summary" %}</h1>
                             <i class="fa fa-code-fork" ng-class="showRelevance ? 'hq-icon' : ''"></i>
                             &nbsp;{% trans "Show Display Conditions" %}
                         </button>
+                        <button type="button" class="btn btn-default"
+                                ng-click="showComments = !showComments"
+                                ng-class="{ 'active': showComments }">
+                            <i class="fa fa-quote-left" ng-class="showComments ? 'hq-icon' : ''"></i>
+                            &nbsp;{% trans "Show Comments" %}
```

Comment:
What would you think of defaulting this to on? Don't feel strongly about it, but I started playing with this locally and was surprised they weren't automatically part of the app summary.